pkg/rbd/controllerserver.go

```diff
@@ -64,7 +63,7 @@ func (cs *ControllerServer) LoadExDataFromMetadataStore() error {
 
 	snap := &rbdSnapshot{}
 	// nolint
-	cs.MetadataStore.ForAll("csi-rbd-(.*)-snap-", snap, func(identifier string) error {
+	cs.MetadataStore.ForAll("csi-rbd-snap-", snap, func(identifier string) error {
```

Comment:
I am not sure whether this/previous naming format has any significance in the backend cluster

Reply:
this is to retrieve all values starts with `csi-rbd-snap` from configmap

Reply:
Wrong identifier used here. Just revert to what it was before and it should be fine.

Reply:
@gman0  current RBD volume key is `csi-rbd-vol-`
to keep the key similar to the rbd volume i made  below change
`snapshotID := "csi-rbd-snap-" + rbdVolume.VolName + "-" + snapName`

let me know do  i need to revert back  the changes?

Reply:
right...but `cs.Metadata.ForAll()` matches only those configmaps that contain `csi-rbd-snap-`
in their name. This needs to be replaced with `csi-rbd-(.*)-snap-` (as it was before).

Reply:
@gman0 reverted back the changes PTAL